README.md

```diff
@@ -0,0 +1,83 @@
+  - Capabilites can be made up of bundled packages or not.
+  - 'black box' is a special type of capability that comes from a vendor, usually with an implied platform requirement.
+  - Some capabilities genereate paper, others just enable other capabilities.
+
+### Implementations
+  - Implementations (imps) are a language-specific, instance of a capablity.
+  - Implementations must be added to a platform by a resource
+  - Implementations have a complexity score that must be maintained by a resource with a comparable score to not stop working.
+
+### Resources
+  - Resources can be either an automation or a human or a combination of both, automations can augment humans.
+
+### Automations
+  - An automation is just a capability (a sub-resource)  that must be maintained by another resource, usually requiring fewer resources.
+
+### Humans
+  - Humans have capability complexity scores. Not having a high enough score means it'll take longer to instanciate a capability, as Research will need to be done.
+  - Humans come with an inital cost and an upkeep, automations are maintained like any other capability.
+  - Upkeep comes in the form of paper.
+  - Sales and Marketing Humans are a special kind of Human that will increase Users per Capability.
+
+### Research
+  - If no resource has a capability at the level required to instantiate something, they can either spend time (with the associated upkeep spend) to research it or you can acquire another Resource with that capability.
+  - Research will increase the capability score of that Human permanently. Humans with higher capabilities have higher upkeep requirements, and may leave you for greener pastures.
+
+### Culture
+  - Culture will affect how your capabilities are instantiated
+  - Some cultures are mutually exclusive with others.
+  - Cultures will decide which Humans and Automations you need to build an maintain Capabilities
+  - Examples include (arch/vendor/os/host : monoculture/polyculture, package culture, hero culture, crunch time, agile, waterfall, flat hiearchy, silos, cult/"family", open vs. closed florr plan, 9-5, rowe, remote or local, contractor culture, independent vs vc backed, hipster vs. new hotness, science lab, optimize-for-happiness, retalitory culture, outsource culture, etc)
+
+### Initial Conditions
+  - The game starts with one resource (you) with a random, but humble, set of capabilities. You must spend time Researching the capabilities you'd like to instantiate.
+  - You'll have 250,000 paper to buy Implementations and acquire Humans.
+
+### Turns
+  - Instantiations, Upkeep and Paper move around during Turns.
+  - If a capability gives you 10 paper/User you get that every turn for evey User.
+  - If a Human costs you 10,000 paper, that is deducted every turn
+  - If an Implementation has an upkeep of 100, it is deducted every turn.
+  - If an Implementation has an upkeep of 100, it is deducted every turn.
+  - 200 Turns can be considered roughly a year in game time.
```

Comment:
That's a LOT of turns. How long is this game designed to last?

Reply:
It depends on the goal you set. and a turn is just a "clock step" like 60 seconds in real time. The idea is to give direction and the "hires" will work autonomously until they get hung up on what to work on next. we could make it fewer turns, I picked 200 because that's 2000/10 or the number of hours in a year/10